Test-runner isn't recognized after installing it in save mode also.

$ npm run test

> web@1.0.0 test C:\Users\anant\Desktop\CollaborativeCodeEditor
> web-test-runner "test/**/*.test.js" --node-resolve

'web-test-runner' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! web@1.0.0 test: `web-test-runner "test/**/*.test.js" --node-resolve`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the web@1.0.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\anant\AppData\Roaming\npm-cache\_logs\2021-08-21T06_03_35_070Z-debug.log[](url)

--- a/docs/docs/dev-server/overview.md
+++ b/docs/docs/dev-server/overview.md
@@ -28,6 +28,20 @@ Install Web Dev Server:
 npm i --save-dev @web/dev-server
 ```
 
+Then add the following to the `"scripts"` section in `package.json`:
+
+```
+"start": "web-dev-server --node-resolve --open"
+```
+
+Or use the shorthand:
+
+```
+"start": "wds --node-resolve --open"
+```
+
+Note, the examples below assume an npm script is used.
+
 ## Basic commands
 
 Start the server:

--- a/docs/docs/test-runner/overview.md
+++ b/docs/docs/test-runner/overview.md
@@ -28,6 +28,20 @@ Install the web test runner:
 npm i --save-dev @web/test-runner
 ```
 
+Then add the following to the `"scripts"` section in `package.json`:
+
+```
+"test": "web-test-runner test/**/*.test.js --node-resolve"
+```
+
+Or use the shorthand:
+
+```
+"test": "wtr test/**/*.test.js --node-resolve"
+```
+
+Note, the examples below assume an npm script is used.
+
 ## Basic commands
 
 Do a single test run:

